### ISSUE TITLE
new colours

### ISSUE THREAD

**Issue by brianblain** [jazzto/2016#10]
Howard would like to see the site reflecting the new coulour scheme (black, gold and white) Dragan is going to mock up something and run it by him. We should maintain the TD green where we can, Stand by

**Comment by andrewfn** [comment]
Ok, as soon as you have the design, sent it to me and we'll do it.

**Comment by mplmnt** [comment]
Hope to have a mock up for you tomorrow Andrew after I get input from the folks at the office.
I do have bit more of a pressing issue, I am emailing you about now.

**Comment by andrewfn** [comment]
I assume the new colour scheme is to go up asap and not wait till Tuesday?

**Comment by mplmnt** [comment]
It has to be ready for the switch to the regular site on April 18th so a bit more time then the weekend :D

**Comment by andrewfn** [comment]
I am confused. Brian said:

> Andrew can you set up test.jazz.to so we can see how to prioritize artists on the front page and start to test & proof. We go live april 12

Is it April 12 or 18?

**Comment by mplmnt** [comment]
My latest information is that the line up announcement is on Monday April 18th certainly the testing should be done ahead of that. If we have the colour changes ready to go before then that is a ideal. Naturally a test of it is necessary before we go live. In Brian's defence, I think that maybe have been the original go to date and perhaps got pushed back, but I will let him chime in on that.

**Comment by brianblain** [comment]
the 18th was news to me, too - I guess they decided to push back the launch

On Thu, Apr 7, 2016 at 6:04 PM, Andrew Fountain notifications@github.com
wrote:

> I assume the new colour scheme is to go up asap and not wait till Tuesday?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/jazzto/2016/issues/10#issuecomment-207109406

## 

Brian Blain
Managing Editor
Toronto Downtown Jazz
82 Bleecker Street, Toronto, ON  M4X 1L8
blain@tojazz.com

www.torontojazz.com
TD Toronto Jazz Festival  June 23 - July 3, 2016
30th Anniversary

**Comment by mplmnt** [comment]
Just wanted to check in and see if everything is going ok with the files I sent

**Comment by andrewfn** [comment]
It's all good. Susanna is working on it and I have set up a development environment linked to http://test.torontojazz.com/
You should see the changes starting to appear on there in the next 24 hours and when we are done and you have signed off on them we'll push them to the production site.

**Comment by andrewfn** [comment]
Dragan, Susanna is making good progress but needs two things from you:
- a new favicon image (or the original of the new logo, I can make a favicon from that)
- new colour for ordinary links to replace the green (that goldy-yellow is too light on white backgrounds)

**Comment by mplmnt** [comment]
Favicon and colour suggestions sent.
I looked over what has been done so far and it sure has come a long way.
A few things while I am here (understanding this is a work progress):
• I do like the dark gold being used for links, works better then my black suggestion.
• We do need to have black backgrounds for the shows on the homepage (to match our advertising).
  I do however see the need for the grey to add some variety to listings on other pages.
• The left and right side columns should have a dark grey background to somewhat counter the large
  amount of black on the homepage.

**Comment by plousia** [comment]
I was a bit confused by some of these comments until I opened the NewTJFcolours-Homepage.jpg file, which I hadn't done before, and realized that the colours are different to the NewTJFcolours-Homepage.psd I was sent. Not sure how that happened, if they're actually different versions, or if GIMP isn't reading the PSD correctly (I'm inclined to think the latter as the sub-menu isn't even displaying in the PSD). As I can't grab colours accurately from the jpg, can you maybe send me a flattened PDF, or just the grey colour values? Sorry about this.

**Comment by mplmnt** [comment]
Yeah both files were identical... would make sense if it was a GIMP problem.
Who knows what other details were missed but I attached the jpeg for that reason precisely.
I am sending a flat pdf right now. Please acknowledge receipt.

**Comment by plousia** [comment]
Right, that's super annoying. Sorry about that. Can you send the PDF to me at scelso@gmail.com? Not sure if it was sent to Andrew...also I see a favicon was sent somewhere, can that be sent directly to me as well? Thanks!

**Comment by mplmnt** [comment]
On its way to your in-box now Susanna.
Please let me know if you need any other components separately as well.

thanks

**Comment by plousia** [comment]
Thanks Dragan. Could you send the new version of the footer logo as well please? Thanks.

**Comment by mplmnt** [comment]
Got it, sent along with the tiff you requested.

**Comment by andrewfn** [comment]
Just noticed that there's a couple of places on http://jazz.to where there's some of the old green.
This is in a different theme Susanna. Can you find it or do you want me to give you the file location?

**Comment by plousia** [comment]
That's the mobile theme, right? Are we converting the mobile theme colours over as well? Thought we were doing just the desktop site.

**Comment by plousia** [comment]
I think http://test.torontojazz.com is probably pretty close to the mockup; please check through to see if there's any changes needed; in other areas of the site than the homepage I interpreted where needed based on the mockup colours, so please make sure everything looks ok.

**Comment by mplmnt** [comment]
Susanna, the test page looks good, you have done all that I have sent in the mock up.

Brian can we please make sure this is populated with the correct shows for the launch (MainStage, Sony Centre and Koerner Hall). There are some double bills that need combined artist shots so I will email those to your later tonight. 

Andrew the coming soon panel in the header should be taken off for the launch please.

As for the mobile site I haven't sent anything about that as I simply haven't had the time to touch it and the folks at the office didn't see it as a priority but I do agree that it should be done;
The colour change should not be as complex I hope as there are only 3 colours to change (pink, green and purple)
- Menu to a dark grey from green with light grey rollovers
- Change the green everywhere else to black  
- Icons on home page to gold background and black art
- Icons on all other pages on black background
- Footer to black from purple 
- Headline type from pink to black (for concert listings etc)
- Links to a different colour, as the pink no longer works (gotta figure that out somehow)perhaps the dark gold Susanna used on the main page would work.

Susanna let me know if I need to mock this up or if what I said here make sense.

**Comment by plousia** [comment]
That should be enough to go on; I'll email with any questions that come up. If you could send me the new version of the icons that would be great, thanks.

**Comment by mplmnt** [comment]
Susanna no new icons, just changing backgrounds
sent you an email with a mock up.

**Comment by andrewfn** [comment]
I just refreshed the content on the test site. No holes on the front page now.

**Comment by plousia** [comment]
Think the mobile site should be mostly done. Please check through and get back to me with any changes. Dragan, the only icons I think I'll need new versions of are the pin and map icons for concerts. Emailed you with the info. Thanks.

**Comment by brianblain** [comment]
I'm getting some text ("syntax error...") at the top of the screen. B

On Sat, Apr 16, 2016 at 6:17 PM, Susanna Celso notifications@github.com
wrote:

> Think the mobile site should be mostly done. Please check through and get
> back to me with any changes. Dragan, the only icons I think I'll need new
> versions of are the pin and map icons for concerts. Emailed you with the
> info. Thanks.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/jazzto/2016/issues/10#issuecomment-210915994

## 

Brian Blain
Managing Editor
Toronto Downtown Jazz
82 Bleecker Street, Toronto, ON  M4X 1L8
blain@tojazz.com

www.torontojazz.com
TD Toronto Jazz Festival  June 23 - July 3, 2016
30th Anniversary

**Comment by plousia** [comment]
Hmm, it looks fine to me. Can you try hitting refresh and clearing cache if necessary and see if the problem persists?

**Comment by mplmnt** [comment]
No errors I can see, looks good.
Sent you an email with the icons.

Andrew is there any way we can see this with the other concert pages?
Just want to make sure we didn't miss any colours there.

thanks

**Comment by andrewfn** [comment]
Yes, and I can see some green: Try http://jazz.to/artistsm/a
- You can touch the green to explore more artists and concert pages.
- The green "Touch here to select artists" really needs to stand out, and I think it would be missed if it were black like everything else. Is there any other alternative? Gold background maybe?

**Comment by mplmnt** [comment]
Just saw that myself, gold would be a good option there, the grey could work well also as in the calendar selection.

**Comment by plousia** [comment]
Whoops, meant to do that and forgot. Will do later today or tomorrow.

**Comment by brianblain** [comment]
******I cleared cache and the syntax error is still there (but that's on on test.jazz.to  -  it's fine on jazz.to)  B

**Comment by andrewfn** [comment]
That explains it. We're not using test.jazz.to so you can ignore it.
- I'm planning to switch over to the new colours on Sunday afternoon (to give us time to check for problems)
- But not go live with the concerts until Monday morning -- what time should we go live?

**Comment by brianblain** [comment]
Not sure about exact time to go live. Stand by.

**Comment by plousia** [comment]
New concert icons added and artist select button colour changed.

**Comment by andrewfn** [comment]
The new theme is live. Please check it out Susanna & Dragan.

**Comment by mplmnt** [comment]
Concert listings - can the concert titles/artist names be gold? 
I think it is necessary to have them pop from the rest of the information.

Menu - once the dropdown appears the title for that section disappears completely. 
Can we have it still be displayed? In rollover colour.

**Comment by andrewfn** [comment]
Does the production site not match the test site? They look the same to me. Or is this a new change in which case Susanna will have to do it.

**Comment by mplmnt** [comment]
Just a couple of things I did not notice before. 
Would be nice if we could get those changed at some point.

**Comment by plousia** [comment]
I've made the concert titles/artist names gold, Andrew's moving it live as I type this.

As for the menu, you are correct. I have tried to fix that but literally can't find a hook to do so using CSS. If you notice on the previous theme, it's the same behaviour: the link is reverting to its normal, non-hover colour. In the previous theme it kind of works because the link is a darker green than the background colour. In this theme it doesn't work because both the default link colour and the hover background colour are black.

When we originally did this site, the plugin we used for the menu didn't have a drupal module, so we integrated the code manually. There's now a Drupal module and the code has advanced quite a bit, and I believe now there are CSS hooks that would allow me to correct this. What I would suggest we should do is add the module and redo the menus using that. If we get the go-ahead, I can start on that.

**Comment by mplmnt** [comment]
Great to see the gold in the concert listings, lets hold on to doing anything to the menu for now.

**Comment by mplmnt** [comment]
Saw that we should also update the icon that displays when app is added to device.
Emailed a square logo we can use for that.
